<launch>
    <param name="step_size"               value="0.05"/>
    <param name="max_x"                   value="0.65"/>
    <param name="min_x"                   value="0"/>
    <param name="start_x"                 value="0"/>
    <param name="max_y"                   value="0.65"/>
    <param name="min_y"                   value="0"/>
    <param name="start_y"                 value="0"/>
    <param name="startup_group_name"      value="arm_with_torso"/>
    <param name="runtime_group_name"      value="arm"/>
    <param name="start_pose_joint_names"  value="[torso_lift_joint, bellows_joint, head_pan_joint, head_tilt_joint, shoulder_pan_joint, shoulder_lift_joint, upperarm_roll_joint, elbow_flex_joint, forearm_roll_joint, wrist_flex_joint, wrist_roll_joint]"/>
    <param name="start_pose_joint_states" value="[0.35, 0.15, 0.0, 0.85, -1.5, -1.0, -2.3, -2.1, 2.4, 0.7, 0.24]"/>

    <node name="discretized_movement" pkg="discretized_movement" type="movement_server" output="screen"/>
</launch>
